fastestimator/util/latex_util.py

```diff
@@ -129,3 +129,51 @@ def __init__(self,
             data.append(bold(escape_latex(name)) if bold_name else escape_latex(name))
         data.append(NoEscape("}}}"))
         super().__init__(data=data)
+
+
+class IterJoin(Container):
+    """A class to convert an iterable to a latex representation.
+
+    Args:
+        data: Data of the cell.
+        token: String to serve as separator among items of `data`.
+    """
+    def __init__(self, data: Iterable, token: str):
+        super().__init__(data=data)
+        self.token = token
+
+    def dumps(self) -> str:
+        """Get a string representation of this cell.
+
+        Returns:
+            A string representation of itself.
+        """
+        return dumps_list(self, token=self.token)
+
+
+class WrapText(LatexObject):
+    """A class to convert strings or numbers to wrappable latex representation.
+
+    This class will first convert the data to string, and then to a wrappable latex representation if its length is too
+    long. This fixes an issue which prevents the first element placed into a latex X column from wrapping correctly.
+
+    Args:
+        data: Input data to be converted.
+        threshold: When the length of `data` is greater than `threshold`, the resulting string will be made wrappable
+    """
+    def __init__(self, data: Union[str, int, float], threshold: int):
+        assert isinstance(data, (str, int, float)), "the self.data type needs to be str, int, float"
```

Comment:
sorry to keep dragging this out, but the doc string needs a Raises section for the assertion error. See fe.backend.abs for an example